plugins/Diagnostics/lang/en.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Diagnostics": {
+        "ConfigFileTitle": "Config file",
+        "ConfigFileIntroduction": "Here you can view the Piwik configuration. If you are running Piwik in a load balanced environment the page might be different depending from which server this page is loaded. Rows with a different background color are changed config values that are specified for example in the %s file.",
+        "HideUnchanged": "If you want to see only changed values you can %shide all unchanged values%s.",
```

Comment:
Unrelated to this PR, but I would suggest to always use numbered placeholders like `%1$s` instead of `%s` in the future. There were already some issues created on transifex, were the order would have needed to be changed.
Might not be the case here, but I guess its better to always do it. (Would love to change that for existing keys aswell, but that would invalidate too much existing translations)